pwndbg/commands/klookup.py

```diff
@@ -12,21 +12,35 @@
 parser.add_argument("symbol", type=str, help="Address or symbol name to lookup")
 
 
+def parse_to_addr(v: str) -> int:
+    if v.startswith("0x"):
+        return int(v[2:], 16)
+    try:
+        return int(v, 16)
```

Comment:
We don't need the first if:
```py
In [1]: int('0x1234', 16)
Out[1]: 4660

In [2]: int('1234', 16)
Out[2]: 4660
```

Reply:
This also means that `klookup <some int in base 10>` will be interpreter as base 16, is that ok?

```py
In [3]: 0xffff1234112
Out[3]: 17591936696594

In [4]: int('17591936696594', 16)
Out[4]: 6571889286276500
```

Reply:
changed to int(x, 0)